packages/react-components/react-combobox/stories/src/Combobox/ComboboxVirtualizer.stories.tsx

```diff
@@ -62,7 +64,9 @@ export const ComboboxVirtualizer = (props: Partial<ComboboxProps>) => {
 ComboboxVirtualizer.parameters = {
   docs: {
     description: {
-      story: 'A Combobox can use Virtualizer to display a large number of options.',
+      story:
+        'A Combobox can use Virtualizer to display a large number of options\n' +
+        `To manually control the maxHeight of the listbox, refer to the [positioning autoSize property](https://react.fluentui.dev/?path=/docs/concepts-developer-positioning-components--default#anchor-to-target)`,
```

Comment:
It would be nice to also add this link to the `Combobox` props table.

![msedge_ooqyv653ZK](https://github.com/user-attachments/assets/6a7d2b71-d4dd-44d2-a605-20b003c85fff)